shared/settings/feedback-container.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+import React, {Component} from 'react'
+
+import {HeaderHoc} from '../common-adapters'
+import Feedback from './feedback'
+import logSend from '../native/log-send'
+import {connect} from 'react-redux'
+import {compose, withState, withHandlers} from 'recompose'
+
+import type {Dispatch} from '../constants/types/flux'
+
+const FeedbackWrapped = compose(
+  withState('sendLogs', 'onChangeSendLogs', true),
+  withHandlers({
+    onSendFeedbackContained: ({sendLogs, feedback, onSendFeedback}) => () => onSendFeedback(feedback, sendLogs),
+  })
+)(Feedback)
+
+type State = {
+  sentFeedback: boolean,
+  feedback: ?string,
+}
+
+class FeedbackContainer extends Component<void, {}, State> {
```

Comment:
I find this form more compact if your constructing just the state to fixed non-prop values

```
class FeedbackContainer extends Component<void, {}, State> {
state: {
   sentFeedback: false,
  feedback: null,
}

// no constructor
```